contracts/Comptroller.sol

```diff
@@ -1131,8 +1136,7 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
             CToken cToken = allMarkets_[i];
             if (markets[address(cToken)].isComped) {
                 Exp memory assetPrice = Exp({mantissa: oracle.getUnderlyingPrice(cToken)});
-                Exp memory interestPerBlock = mul_(Exp({mantissa: cToken.borrowRatePerBlock()}), cToken.totalBorrows());
-                Exp memory utility = mul_(interestPerBlock, assetPrice);
+                Exp memory utility = mul_(assetPrice, cToken.totalBorrows());
```

Comment:
so interest rate now a function of totalBorrows * assetPrice?

Reply:
Yes, discussion here https://compound.comradery.io/post/1762 under "Potential Solution 3"-- the borrow rate multiplier effectively was pushing markets toward the poles. This patch may alleviate that pressure.

Reply:
Is there an ETA for this so people can't game the system anymore?

Reply:
Moving from interest per block to total borrows also has the upside of being a bit more straightforward to implement and to reason about, as evidenced here 👍